### PR TITLE
Add inventory w/ configurable items

### PR DESCRIPTION
**Pull request by mhartenbower**

### CODE REVIEW

File: item.py
Comment (jeethsuresh) [inline item.py:1]: This is just semantics but maybe we should call these "towers"? It's fine for now but I wanna get your temperature on changing it in future

**Review by jeethsuresh — APPROVED** [review]
LGTM! I'm gonna give you merge access so merge whenever you're ready :)

**Comment by jeethsuresh** [comment]
Actually I wanna toy with this before bed so I'm gonna merge

File: item.py
Comment (mhartenbower) [reply item.py:1]: yup, item was just a placeholder

**Pull request merged**
